sycl/test-e2e/Graph/RecordReplay/buffer_copy_host2target_2d.cpp

```diff
@@ -1,7 +1,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 // Extra run to check for leaks in Level Zero using UR_L0_LEAKS_DEBUG
-// RUN: %if ext_oneapi_level_zero %{env UR_L0_LEAKS_DEBUG=1 %{run} %t.out 2>&1 | FileCheck %s %}
+// RUN: %if level_zero %{env UR_L0_LEAKS_DEBUG=1 %{run} %t.out 2>&1 | FileCheck %s %}
```

Comment:
2Graph team: I'd suggest to modify your RUN lines to use `FileCheck %s --implicit-check-not=LEAK` and drop line 6 here and in other tests.

Reply:
Thanks for the tip, we can do this as a follow-up PR